features/0547-invite-action-protocol/README.md

```diff
@@ -0,0 +1,132 @@
+# Aries RFC 0547: Invite Action Protocol
+
+- Authors: [Daniel Hardman](daniel.hardman@gmail.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-09-30
+- Status Note: newly submitted for community consideration, based on a use case developed by the credit union industry in the United States. Implements a concept first called "please play a role" that was discussed on community calls in mid 2020 in conjunction with the [Out of Band protocol](../0434-outofband/README.md).
+- Start Date: 2020-09-30
+- Tags: [feature](/tags.md#feature), [protocol](/tags.md#protocol)
+- URI: https://didcomm.org/invite-action/%VER 
+
+## Summary
+
+Provides a mechanism for one [agent](../../concepts/0004-agents/README.md) to invite another to take a particular action. The action is described in a way that can be automated. It may be accompanied by supporting data. The invited agent may or may not trust the inviter, and may choose to react in various ways -- ignore the invitation, accept it, defer a decision, etc.
+
+## Motivation
+
+DIDComm protocols typically make a strong assumption about who is the "first mover" -- the role that must emit the protocol's initial message. For example, in [RFC 0037: Present Proof Protocol](../0037-present-proof/README.md), the `verifier` starts the interaction by sending a `request-presentation` message; the `prover` can't go first.
```

Comment:
> the `prover` can't go first

The Holder can initiate by sending `propose-presentation`.

Reply:
You're right. I didn't study the choreography diagram carefully enough.

I'll come up with another example.

Thanks for the catch.

Reply:
Fixed.